packages/wonder-blocks-tokens/src/theme/semantic/semantic-color-thunderblocks.ts

```diff
@@ -726,7 +726,7 @@ export const semanticColor = mergeTheme(defaultSemanticColor, {
         default: {
             border: core.border.neutral.default,
             background: surface.primary,
-            foreground: core.foreground.neutral.default,
+            foreground: core.foreground.neutral.strong,
```

Comment:
I can't remember why I changed it to `neutral.default` in a previous PR 😅  `neutral.strong` is what's used for the input field value! 
![image](https://github.com/user-attachments/assets/c2c1a57f-874f-4d21-84fb-ab48ce1792e2)